.github/workflows/diagram-cache.yml

```diff
@@ -25,6 +25,17 @@ jobs:
             --output ./diagram_cache
             --index
             --docker ghcr.io/dsd-dbs/capella-dockerimages/capella/base:{VERSION}-${{ env.CAPELLA_DOCKER_IMAGES_DROPINS }}-dropins-${{ env.CAPELLA_DOCKER_IMAGES_REVISION }}
+        - name: Push diagram cache to orphan branch
+          run: |
+            DERIVED_BRANCH_NAME="$(echo $COMMIT_BRANCH-diagram-cache)"
+            git switch "$DERIVED_BRANCH_NAME" || git switch --orphan "$DERIVED_BRANCH_NAME"
+            git reset
+            git add diagram_cache
+            if git diff --cached --exit-code &> /dev/null || [[ "$PUSH_DIAGRAM_CACHE" == 0 ]]; then exit 0; fi
+            git config --local user.email "github-actions[bot]@users.noreply.github.com"
+            git config --local user.name "github-actions[bot]"
```

Comment:
I'd change it to another name which doesn't correlate with Github. Something like `capellambse[bot]`.